src/plugins/mer/merbuildconfigurationwidget.ui

```diff
@@ -54,6 +54,62 @@
      </property>
     </widget>
    </item>
+   <item row="2" column="0" colspan="2">
+    <widget class="QGroupBox" name="signPackagesGroupBox">
```

Comment:
It would be more natural if `Additional sfdk configuration:` came at the very end. I would also try to go without (visible) group box. Just something like:

```
RPM spec file:                 [                                            ] [Browse...]
[X] Sign packages
        Signing user:          [                                           v]
        Passphrase file:       [                                            ] [Browse...]
Additional sfdk configuration: [                                            ]